src/pages/workspace/accounting/qbo/QuickbooksImportPage.tsx

```diff
@@ -0,0 +1,109 @@
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_CHART_OF_ACCOUNTS.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.enableNewCategories),
+            title: enableNewCategories,
+            pendingAction: pendingFields?.enableNewCategories,
+        },
+        {
+            description: translate('workspace.qbo.classes'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_CLASSES.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncClasses),
+            title: syncClasses,
+            pendingAction: pendingFields?.syncClasses,
+        },
+        {
+            description: translate('workspace.qbo.customers'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_CUSTOMERS.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncCustomers),
+            title: syncCustomers,
+            pendingAction: pendingFields?.syncCustomers,
+        },
+        {
+            description: translate('workspace.qbo.locations'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_LOCATIONS.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncLocations),
+            title: syncLocations,
+            pendingAction: pendingFields?.syncLocations,
+        },
+        {
+            description: translate('workspace.qbo.taxes'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_TAXES.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncTaxes),
+            title: syncTaxes,
+            pendingAction: pendingFields?.syncTaxes,
+        },
+    ];
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <FeatureEnabledAccessOrNotFoundWrapper
+                policyID={policyID}
+                featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            >
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    shouldEnableMaxHeight
+                    testID={QuickbooksImportPage.displayName}
+                >
+                    <HeaderWithBackButton title={translate('workspace.qbo.import')} />
+                    <ScrollView contentContainerStyle={styles.pb2}>
+                        <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.qbo.importDescription')}</Text>
+                        {sections.map((section) => (
+                            <OfflineWithFeedback
+                                key={section.description}
+                                pendingAction={section.pendingAction}
+                            >
+                                <MenuItemWithTopDescription
+                                    title={quickbooksOnlineConfigTitles[`${section.title ?? false}`]}
```

Comment:
I think I'm missing something, shouldn't this be 
```suggestion
                                    title={quickbooksOnlineConfigTitles[`${section.title ?? quickbooksOnlineConfigTitles[CONST.INTEGRATION_ENTITY_MAP_TYPES.NOT_IMPORTED]}`]}
```

which would allow us to get rid of `quickbooksOnlineConfigTitles['true']` and `quickbooksOnlineConfigTitles['false']`